src/index.js

```diff
@@ -48,6 +48,7 @@ import sv from 'react-intl/locale-data/sv';
 import th from 'react-intl/locale-data/th';
 import tr from 'react-intl/locale-data/tr';
 import uk from 'react-intl/locale-data/uk';
+import uz from 'react-intl/locale-data/uz';
 import vi from 'react-intl/locale-data/vi';
 import zh from 'react-intl/locale-data/zh';
 import zu from 'react-intl/locale-data/zu';
```

Comment:
Seems like we should update the commented out `locales =` code immediately below, or else delete it.